File: public/example/example_2.java
using System;

public class SimpleCSharpExample
{
    static int Square(int num)
    {
        return num * num;
    }

    static bool CheckEvenOdd(int num)
    {
        return num % 2 == 0;
    }

    public static void Main()
    {
        int num1 = 5;
        int result1 = Square(num1);

        int num2 = 10;
        CheckEvenOdd(num2);
    }
}

File: public/example/example_2.go
package main

import "fmt"

func square(num int) int {
    return num * num
}

func checkEvenOdd(num int) bool {
    return num%2 == 0
}

func main() {
    num1 := 5
    result1 := square(num1)

    num2 := 10
    checkEvenOdd(num2)

    fmt.Println(result1)
}
